documentation/smg/14_documentation.md

```diff
@@ -0,0 +1,45 @@
+# Documentation
+
+This document will outline the process surrounding the documentation currently housed in the `horace-docs` repository. It assumes a basic knowledge of Restructured Text (ReST) format (https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html) especially the subset of the sphinx-doc toolset (https://www.sphinx-doc.org/en/master/)
```

Comment:
Worried that ReST will be confusing for those thinking about the REST/SOAP API alternatives. The linked document refers consistently to reStructured text with no abbreviation. Suggesting that we keep the capitalization of the full name (i.e. Sem-camel-case) and use the rst of the suffix (asis or all-caps, at your preference) as the abbreviation.

Reply:
Now looking at Sphinx and they do use reST (but note the capitalization.) It may still be worth alerting the reader to the ambiguity (" ... this is reST, the restructured text markup, not REST the API.")

Reply:
Updated throughout

Reply:
From talking to Nick, all this process was decided and documented in ADR 2 of pace devs. It could be useful to record that here.